### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by uu59**

### CODE REVIEW

File: test/embulk/guess/test_query_string.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+              ]
+            }
+          }
+          assert_equal(expected, actual)
+        end
+      end
+
+      private
+
+      def sample_lines_1
```
Comment (yoshihara) [inline test/embulk/guess/test_query_string.rb:44]: What does `_1` means?

File: test/embulk/parser/test_query_string.rb [hunk 2]
```diff
@@ -0,0 +1,116 @@
+require "prepare_embulk"
+require "embulk/parser/query-string"
+require "embulk/data_source"
+
+module Embulk
+  module Parser
+    class QueryStringPluginTest < Test::Unit::TestCase
+      class TestParse < self
+        def test_parse_line_without_options
```
Comment (yoshihara) [inline test/embulk/parser/test_query_string.rb]: Class name specifies that these tests check `parse` method already, so method names don't need `parse` term I think.

File: test/embulk/guess/test_query_string.rb (hunk 1 above)
Comment (uu59) [inline test/embulk/guess/test_query_string.rb:44]: Added `_2`, each number has some different pattern.

Comment (yoshihara) [inline test/embulk/guess/test_query_string.rb:44]: Hmm..I'll think more meaningful names:) (I want each name to specify content of data)

**Comment by yoshihara** [comment]
LGTM

**Pull request merged**